sdk/openai/Azure.AI.OpenAI/tests/OpenAIInferenceTests.cs

```diff
@@ -94,5 +110,32 @@ public void RequestFailedExceptionTest()
             var exception = Assert.ThrowsAsync<RequestFailedException>(async () => { await client.GetCompletionsAsync("BAD_DEPLOYMENT_ID", completionsRequest); });
             Assert.AreEqual(404, exception.Status);
         }
+
+        [RecordedTest]
+        public async Task StreamingCompletionsTest()
```

Comment:
We need Asserts in tests, right?

Reply:
Absolutely; I originally rushed this draft out right before my few days away and should have made that more clear in the description. The tests have asserts now (and quite a few of them!).